routing/pathfind.go

```diff
@@ -595,16 +588,17 @@ func findPath(g *graphParams, r *RestrictParams, source, target route.Vertex,
 			// the node on the _other_ end of this channel as we
 			// may later need to iterate over the incoming edges of
 			// this node if we explore it further.
-			channelSource, err := edgeInfo.FetchOtherNode(
-				tx, pivot[:],
-			)
+			channelSource, err := edgeInfo.OtherNodeKeyBytes(pivot[:])
 			if err != nil {
 				return err
 			}
 
+			var source [33]byte
+			copy(source[:], channelSource)
```

Comment:
I would prefer just changing the return type of `OtherNodeKeyBytes` to `[33]byte` if possible (and no significant perf impact)

Reply:
I agree

Reply:
Done